spec/support/matchers/table_matchers.rb

```diff
@@ -1,3 +1,5 @@
+# FIXME - this seems to be very slow matcher (features/admin/reports_spec.rb for products and inventory reports)
+# need to fix it before using it
```

Comment:
Is this only slow for the new reports? Or is it slow for the old version as well?